build.sbt

```diff
@@ -8,7 +8,8 @@ version       := "0.1"
 
 scalaVersion  := "2.11.7"
 
-scalacOptions := Seq("-unchecked", "-deprecation", "-encoding", "utf8")
+scalacOptions := Seq("-unchecked", "-deprecation", "-encoding", "utf8",
+  "-Xmax-classfile-name", "100")
```

Comment:
I'm hesitant to include this without first figuring out why only your computer needs this change.